### PR TITLE
PR for Image Component

### PR DESCRIPTION
**Pull request by manojasarsa**
Added Responsive Image and Round Image.

### CODE REVIEW

**Review by kiranpratyush — APPROVED** [review]
We can format our HTML to look beautiful

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.img_responsive {
+    max-width: 100%;
+    height: auto;
+}
+
+.img_round {
+    border-radius: 50%;
+}
```
Comment (adidoshi) [inline components/image/image.css:16]: Hey, just a suggestion if height & width are specified of same size, then it would look like a perfect circle.

**Review by adidoshi — APPROVED** [review]
Rest looks good!!

File: components/image/image.css (hunk 1 above)
Comment (adidoshi) [reply components/image/image.css:16]: Like this one:
https://res.cloudinary.com/splashcloud/image/upload/v1644239863/check/check0_vmxtbt.png

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import url("/docs/main.css");
+
+.image_container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
```
Comment (vaibhav18matere) [inline components/image/image.css:6]: can make a separate class like `flex-start`

File: components/image/image.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            <div class="image_container">
+                <img src="/assets/concert.jpg" alt="" class="img_responsive">
+            </div>
+
+            <div class="display_code">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14.5px&lh=147%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522img_responsive%2522%2520alt%253D%2522...%2522%253E"
+                    style="width: 456px; height: 95px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
```
Comment (vaibhav18matere) [inline components/image/image.html:99]: use prettier so that code will look aligned to content

**Review by vaibhav18matere — APPROVED** [review]

File: components/image/image.css (hunk 1 above)
Comment (manojasarsa) [reply components/image/image.css:16]: Thanks Aditya, will make changes!

**Pull request merged**